models/stg/google_sheet/schema.yml

```
@@ -113,4 +113,54 @@ sources:
             tests:
               - unique
           - name: member_image_link
+            data_type: STRING
+      - name: party
+        description: >
+          Which party each member belonged to in which parliament. If parliament is blank, then member was in the same party for all parliaments they were in.
+        external:
+          options:
+            format: GOOGLE_SHEETS
+            uris: ['https://docs.google.com/spreadsheets/d/1Wk_PDlQbWWViTV9NmDPsXwu54TD96ltEycLAKOHe1fA/']
+            sheet_range: members_party
+            skip_leading_rows: 1
+        columns:
+          - name: member_name
+            data_type: STRING
+            tests:
```

```suggestion
            data_tests:
```

from dbt 1.8 and onwards, there's a deprecation warning for `tests`. there's unit tests and data tests now. so we have to refer to them as data tests.

i've have to edit all of them when i bump the dbt version.